### DISCUSSION TITLE
can't find STM32CubeIDE in dmenu

### DISCUSSION THREAD

**Discussion by selimkeles** [i3/i3#5086] [Q&A]
As the title says I am new to i3wm and i have installed STM32CubeIDE from bash script they have provided in their website. The problem is dmenu can't find that application. I have faced this problem for flatpak apps but this post https://gist.github.com/curioswati/668e9e120ddd4b6f8d07dc28b5780d22 helped me to solve problem.

Problem is I couldn't find binary files for stm32cubeide. I used commands like "find" and "whereis" but couldn't locate even a single file about stm32. I opened install-bashscript in vim and tried to watch where it creates installation files but couldn't make it either.

**Comment by Airblader** [comment]
If you can't figure out yourself what the entry point of the application is, I would ask the developers of the application.
